_docs/getting-started/centos.md

```diff
@@ -28,7 +30,11 @@ For Caffe2 with CUDA 8 and CuDNN 7 support:
 conda install -c caffe2 caffe2-cuda8.0-cudnn7
 ```
 
-To see what packages these pre-built binaries are built against, see `conda/cuda/meta.yaml` for the CUDA builds and `conda/no_cuda/meta.yaml` for the CPU only build. If you want different libraries then you must build from source.
+> This does NOT include libraries that are necessary to run the tutorials, such as jupyter. See the [tutorials](https://caffe2.ai/docs/tutorials) page for the list of required packages needed to run the tutorials.
+
+NOTE: This will install Caffe2 and all of its required dependencies into the current conda environment. We strongly suggest that you create a new conda environment and install Caffe2 into that. A conda environment is like its own python installation that won't have library version problems with your other conda environments. You can learn more about conda environments [here](https://conda.io/docs/user-guide/tasks/manage-environments.html).
```

Comment:
Nit: A conda environment is like its own python installation that won't have library version problems with your other conda environments -> A conda environment is like a separate python install and so won't have library version problems with your other conda environments